theme/docs/conf.py

```diff
@@ -3,7 +3,7 @@
 
 # -*- coding: utf-8 -*-
 #
-# Daml SDK documentation build configuration file, created by
+# Daml documentation build configuration file, created by
```

Comment:
These are default values currently overridden by the `config.py` files coming from the daml repo bundle. Still, I thought updating them made sense.